examples/nats-streaming-start-position/publisher.out

```diff
@@ -0,0 +1,13 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file, and execute the `ballerina run` command.
+$ ballerina run publisher.bal
+Subject : demo
+[ballerina/nats] Connection established with server nats://localhost:4222
```

Comment:
```suggestion
[ballerina/nats] Connection established with the NATS server://localhost:4222
```

Reply:
This is a log from the client.
nats://localhost:4222 is the url.